model/restful/utils/task.py

```diff
@@ -43,105 +43,233 @@ def getForwardHeaders(request):
     return headers
 
 
-def run_task(service_endpoint):
+def run_task(service_name, service_endpoint):
     headers = getForwardHeaders(request)
```

Comment:
We are already have the same value for headers as defined here, also we are importing headers from wsgiref, don't they conflict with each other? And do we really need wsgiref?

Reply:
I didn't make changes to the header part... If I remember correctly, this was done to forward jaeger tracing headers through the set of microservices